packages/design/src/Form/index.tsx

```diff
@@ -115,7 +115,7 @@ export default function Form({
                 <FormContents context={context} prompt={prompt} />
               </form>
             ) : (
-              <div>
+              <div className="formContentWrapper">
```

Comment:
I added that class name so that I can locate that component once I navigate away from it. With these components only having utility class names applied to them, it's hard to differentiate from each other when doing a search.